Breakout/GameModifiers/GameModifierStorage.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Breakout.GameModifiers.Hazard;
+using Breakout.GameModifiers.PowerUps;
+using Breakout.Hazard;
+
+namespace Breakout.GameModifiers;
+
+/// <summary>
+/// Static class that stores all the power up types.
+/// </summary>
+public static class GameModifierStorage
+{
+    /// <summary>
+    /// A collection of all the power up types.
+    /// </summary>
+    private static readonly List<IGameModifier> PowerUps = new();
+    /// <summary>
+    /// A collection of all the hazards.
+    /// </summary>
+    private static readonly List<IGameModifier> Hazards = new();
+
+    
+    /// <summary>
+    /// Static constructor that adds all the power up types to the collection.
+    /// </summary>
+    static GameModifierStorage()
```

Comment:
Har du sørget for at den smider de rigtige under hazards og powerups? Havde i forvejen lavet 2 separate storages, virker ikke optimalt at have dem i den samme